frontend/my-app/src/components/DonutChart.js

```diff
@@ -1,17 +1,14 @@
 import React, {useEffect, useState} from 'react';
 import {Doughnut} from 'react-chartjs-2';
 import {Chart as ChartJS} from "chart.js/auto"
+import {Card} from 'react-bootstrap'
+import 'bootstrap/dist/css/bootstrap.min.css';
 
 const DonutChart = (props) => {
-
-    //should get this from automated meeting number, hardcoded for now
-    const lastGenMeetingNum = 2;
-
-
     const [chart, setChart] = useState();
 
     const fetchData = async () => {
-        await fetch("http://localhost:5000/DonutData?type=" + props.type + "&last=" + lastGenMeetingNum, {
+        await fetch("http://localhost:5000/DonutData?type=" + props.type, {
```

Comment:
You can do string interpolation here like

\`http://localhost:5000/DonutData?type=${props.type}`